sources/platform/integrations/langchain.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: LangChain
+title: 🦜🔗 LangChain
```

Comment:
That is a good point, I should strive to standarize it. In this case I went with `integration` since it was through third party, but maybe we should go with one approach for all of them, either with integration or without, let me think about it.

Reply:
I removed integration from title in each case except Telegram, since this one is through 3rd party. I think this approach works best as each doc is already under Integrations folder in the sidebar.